Kr1/CheckSum.Src/CheckSumCalculator.cs

```diff
@@ -0,0 +1,91 @@
+using System.Security.Cryptography;
+
+namespace CheckSum;
+
+public class CheckSumCalculator
+{
+    public static byte[] Calculate(string path)
+    {
+        FileAttributes attr = File.GetAttributes(path);
+        if ((attr & FileAttributes.Directory) == FileAttributes.Directory)
+            return HashDirectory(new DirectoryInfo(path));
+        else
+            return HashFile(new FileInfo(path));
+    }
+
+    public static async Task<byte[]> CalculateAsync(string path)
+    {
+        FileAttributes attr = File.GetAttributes(path);
+        if ((attr & FileAttributes.Directory) == FileAttributes.Directory)
+            return await HashDirectoryAsync(new DirectoryInfo(path));
+        else
+            return await HashFileAsync(new FileInfo(path));
+    }
+
+    private static byte[] HashDirectory(DirectoryInfo info)
+    {
+        using (MemoryStream stream = new())

```

Comment:
```suggestion
        using MemoryStream stream = new();
```
И дальше без лишних отступов